### ISSUE TITLE
Suggestion: Make hiding miner windows optional

### ISSUE THREAD

**Issue by Haydenman2** [MultiPoolMiner/MultiPoolMiner#1293]
Hello! I downloaded the latest version, and I like the online monitoring, but I don't like how it hides the mining windows.

I think that this feature should be optional and enabled by default, since myself and probably some others like seeing the active running windows of the individual miners. Maybe with a -showminers flag in the start script.

Thank you!

**Comment by alucard20724** [comment]
i agree... but it's already been discussed.  they're going to add it to version 2.7.3 release in march.   I'll be using version 2.7.1 until it gets back into the code.

**Comment by UselessGuru** [comment]
> they're going to close this thread, and then a day later, someone else is going to post about the same thing.

Sad enough people do no use the search function... As soon as the topic moves aawy from the first page it is basically 'ignored'

**Comment by angelics** [comment]
i hated the hidden windows so, i modified a local version from pre-job.
i made it public after aaron implemented log reader.
https://github.com/angelics/SephMiner

**Comment by aaronsace** [comment]
I understand that people like to see them but this allows for actual debugging that was not possible before. If you look inside the logs folder then you will see all the miner windows are there. 'Hate' and 'like' get thrown around but MultiPoolMiner is made to manage the miners and that is what it does.

**Comment by vhatzist** [comment]
Would it be possible to have both the miner windows on screen and their log files at the same time?

**Comment by aaronsace** [comment]
@vhatzist The miner screen is rendered onto the hard disk. 

This is how to show the miner windows 'Reader.ps1': 
start pwsh -noexit -executionpolicy bypass -command "& .\reader.ps1 -log '^(?!MultiPoolMiner_)' -sort '^[^_]*_' -quickstart"

**Comment by paulpoco** [comment]
Can Powershell be configured for 4 terminal window?

Would be cool have the current window we now see now in say the upper left window.  Then each Miner running in the other windows.

This way you wouldn't have the Miner screens popping up all over the place.

An example of multiple terminals in one window is Linux Terminator.
[https://www.tecmint.com/wp-content/uploads/2014/01/Terminator-Emulator.jpeg](url)

**Comment by paulpoco** [comment]
![terminator-emulator](https://user-images.githubusercontent.com/801414/36603293-244aa1d8-186f-11e8-8bd4-f250500ce32f.jpeg)

**Comment by aaronsace** [comment]
Could do but I'm not sure how you get the windows to do that. I think the GUI interfaces that @grantemsley is working on will be able to do that better but this is all going to be v2.8 and later.

**Comment by borzaka** [comment]
Is this shortens my SSD's lifetime?

**Comment by grantemsley** [comment]
@borzaka it's not writing that much data, it won't wear out an SSD anytime soon.  That isn't really much of an issue for typical users with modern SSDs.

**Comment by borzaka** [comment]
Thank you for the info.
I'am afraid because it's constantly writes small txt files 24/7. And after running 1 hour, my Logs folder is 1 MB.

**Comment by grantemsley** [comment]
Most 240GB SSDs won't have any issue until you've written at least 200-300TB, with some of them being good for around a petabyte of writes.  So at 1MB per hour, 24/7, you are looking at around 20-30 years before you might start seeing issues from the SSD wearing out.

SSD wear out isn't really a concern for anything except extremely write heavy environments.

**Comment by borzaka** [comment]
Thank you for clarifying!
